server/src/services/app_import_export.service.ts

```diff
@@ -102,6 +102,7 @@ export class AppImportExportService {
           .where('data_sources.appVersionId IN(:...versionId)', {
             versionId: appVersions.map((v) => v.id),
           })
+          .andWhere('data_sources.scope != :scope', { scope: DataSourceScopes.GLOBAL })
```

Comment:
Can we rename the variable `datasources` assignment here with rather `legacyLocalDataSources`?